tests/composite_pk/tests.py

```diff
@@ -147,6 +147,21 @@ def test_in_bulk(self):
         result = Comment.objects.in_bulk([self.comment.pk])
         self.assertEqual(result, {self.comment.pk: self.comment})
 
+    @unittest.mock.patch.object(
+        type(connection.features), "max_query_params", new_callable=lambda: 10
+    )
+    def test_in_bulk_batching(self, mocked_max_query_params):
+        num_requiring_batching = (connection.features.max_query_params // 2) + 1
+        comments = [
+            Comment(id=i, tenant=self.tenant)
+            for i in range(2, num_requiring_batching + 1)
```

Comment:
I'm starting at 2 because setUpTestData currently supplies one Comment. I don't know if there's a more future-proof pattern for this kind of thing.

Reply:
Deleting the existing comment might be simplest, as it's a fast query either way.